demos/link-prediction/metapath2vec-link-prediction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>Also out of curiosity, I wonder what the performance of Node2Vec (i.e. just ignoring the node types entirely) on this dataset would be.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Hmm yeah hard to say without actually running it, but given that there's 333983 user-friend-&gt;user edges, and only 14476 group-belongs-&gt;user edges, I guess it may not be as effective at finding users that are connected via groups, and instead predict friendships mostly based on other friendships?</p><p> </p>